litmus-portal/frontend/src/views/WorkflowDetails/WorkflowNodeInfo/index.tsx

```diff
@@ -112,16 +110,30 @@ const WorkflowNodeInfo: React.FC<WorkflowNodeInfoProps> = ({
           {/* Button to show Application Details */}
           {data.nodes[pod_name].type === 'ChaosEngine' && embeddedYAMLString && (
             <>
-              <Button
-                disabled={!YAML.parse(embeddedYAMLString).spec.appinfo}
+              <div
+                role="button"
+                onKeyPress={() => {}} // Keyboard Event handler not required for this button
                 onClick={() => setIsAppInfoVisible(!isAppInfoVisible)}
-                style={{ textTransform: 'none' }}
-                className={classes.buttonAlign}
+                className={`
+                  ${
+                    !YAML.parse(embeddedYAMLString).spec.appinfo
+                      ? classes.disabled
+                      : ''
+                  } ${classes.buttonFlex}`}
+                tabIndex={0}
               >
                 {isAppInfoVisible ? (
-                  <KeyboardArrowDownIcon className={classes.icon} />
+                  <img
+                    src="/icons/down-arrow.svg"
+                    alt="down-arrow icon"
+                    className={classes.icon}
+                  />
                 ) : (
-                  <ChevronRightIcon className={classes.icon} />
+                  <img
+                    src="/icons/right-arrow.svg"
+                    alt="right-arrow icon"
+                    className={classes.icon}
+                  />
                 )}
```

Comment:
please use icon button or outlined button from MUI / litmus-ui instead of this much extra code. @Jonsy13 cc: @whitetiger1399 @arkajyotiMukherjee

Reply:
The problem is I needed a simple button, which doesn't have outline or anything.
I tried using button from MUI, but there they have padding for text inside it, but I wanted the text to be aligned to left with padding 0 which wasn't happening even after providing paddingLeft as 0.

Reply:
icon button gives the functionality of a button...but has no outlines...it might have some padding which you can easily overwrite according to your liking @Jonsy13

Reply:
https://material-ui.com/components/buttons/#icon-buttons